tests/test_all_rules.py

```diff
@@ -370,6 +370,11 @@ def test_rule_file_name_tactic(self):
             rule_err_str = '\n'.join(bad_name_rules)
             self.fail(f'{error_msg}:\n{rule_err_str}')
 
+    def test_bbr_in_correct_dir(self):
+        """Ensure that the BBR is in the correct directory."""
+        for rule in self.bbr:
+            self.assertEqual(rule.path.parent.name, 'bbr', f'{self.rule_str(rule)} should be in the bbr directory')
```

Comment:
lets rename the `bbr` directory to `bbr_rules`

Reply:
to also check that _only_ bbr rules are in this directory, you could just iterate all rules and verify it is in `bbr_rules` or `rules` depending on BBR or not.

Reply:
Added this check in `test_bbr_in_correct_dir`